tools/static-assets/skel/package.json

```diff
@@ -2,10 +2,20 @@
   "name": "~name~",
   "private": true,
   "scripts": {
-    "start": "meteor run"
+    "start": "meteor run",
+    "test": "meteor test --driver-package dispatch:mocha-browser",
```

Comment:
I wanted to use `meteortesting:mocha`, but my out-of-box experience was that the server restarted three times and tests didn't run in the browser because the app was "crashing."

Reply:
Right - to avoid this you have to set `TEST_WATCH=1` first. So: `TEST_WATCH=1 meteor test --driver-package meteortesting:mocha`

Reply:
...Which I believe necessitates [`cross-env`](https://www.npmjs.com/package/cross-env) on Windows.

Reply:
Thanks @abernix - and yuck 🙁. Maybe's there another alternative here then (like finding out why `meteortesting:mocha` is setup to work like this). I'm just a bit worried about putting `dispatch:mocha-browser` in, since it has been deprecated.

Reply:
Agree. `meteortesting:*` has been working great overall.  Paging @aldeed for input!

Reply:
I took a quick look - `meteortesting:mocha` requires either the `--once` flag or `TEST_WATCH=1` to be set. Otherwise it does a [`process.exit(0)`](https://github.com/meteortesting/meteor-mocha/blob/1d8eacc1afdcaec5a25eef0416a9a153f4df4d09/package/server.js#L71) (after running the tests once and there are no failures), which the Tool then re-runs until it considers the app to be crashing.

If we just want to show that tests can be run once (via `meteor npm test`), then we can probably get away with using the `--once` parameter (so: `meteor test --once --driver-package meteortesting:mocha`). If we instead want to have things setup so `meteor npm test` runs the tests in watch mode, then maybe we should open a `meteortesting:mocha` PR to make watch mode the default.